main.go

```diff
@@ -42,12 +42,21 @@ func main() {
 
 	rand.Seed(time.Now().UnixNano()) // make sure our random numbers actually are random (but remember, use crypto/rand for anything where strong/reliable randomness is required
 
+	azcopyLogPathFolder := common.GetLifecycleMgr().GetEnvironmentVariable(common.EEnvironmentVariable.LogLocation())     // user specified location for log files
+	azcopyJobPlanFolder := common.GetLifecycleMgr().GetEnvironmentVariable(common.EEnvironmentVariable.JobPlanLocation()) // user specified location for plan files
+
 	// note: azcopyAppPathFolder is the default location for all AzCopy data (logs, job plans, oauth token on Windows)
 	// but both logs and job plans can be put elsewhere as they can become very large
 	azcopyAppPathFolder := GetAzCopyAppPath()
```

Comment:
azcopyAppPathFolder is obsolete imo, the oauth token should just go with azcopyJobPlanFolder

Reply:
perhaps just get rid of it completely, and perform a bit of refactoring to clean it up 😉

Reply:
yes I think so too. If we're going to put OAuth to JobPlanFolder, there isn't a lot of use for the AppPathFolder. I'll make the necessary changes 😎

Reply:
this variable will still be present, but only be local to this function and be used to make azcopyLogPathFolder and azcopyJobPlanFolder if they're empty.

Reply:
changes done. Closing this

Reply:
link to commit - https://github.com/Azure/azure-storage-azcopy/pull/1782/commits/7f67778481dba62323c2636b5ca0e767edf2cf73